pkg/controller/restapi/operation/messaging/models.go

```diff
@@ -0,0 +1,84 @@
+/*
```

Comment:
should double check on whether or not anonymous embedded model structs should be pointers.

Reply:
removed pointers, it didn't hurt spec.